### DISCUSSION TITLE
How to take screenshot of sddm login screen?

### DISCUSSION THREAD

**Discussion by tryprncp** [prasanthrangan/hyprdots#1992] [Q&A] [answered]
How to take screenshot of sddm login screen? I've read some tutorial online but it's specific for xorg. I'm yet to find a workaround for wayland

**Comment by beedell-roke** [comment, marked as answer]
@tryprncp, see [`discuss.kde.org/t/36367/2`](https://discuss.kde.org/t/how-to-screenshot-sddm/36367/2?u=rokejulianlockhart).
